audit/audit.go

```diff
@@ -16,9 +16,10 @@ const (
 	// Required fields: Entity, ACL
 	OpSetPerm Operation = "set-perm"
 
-	// OpSetPromulgated represents the promulgation on an entity.
-	// Required fields: Entity, Promulgated
-	OpSetPromulgated Operation = "set-promulgated"
+	// OpSetPromulgate, OpSetUnPromulgate represent the promulgation on an entity.
```

Comment:
I'd suggest OpPromulgate and OpUnpromulgate (the "Set" is redundant).

    // OpPromulgate represents the promulgation of an entity.
    OpPromulgate = "promulgate"

    // OpUnpromulgate represents the unpromulgation of an entity.
    OpUnpromulgate = "unpromulgate"